assets/js/components/KeyMetrics/MetricsSelectionPanel/SelectionPanelItems.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Selection Panel Items component.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import ChipTabGroup from '../ChipTabGroup';
+
+export default function SelectionPanelItems( {
+	savedItemSlugs = [],
```

Comment:
Given this isn't a required prop and the propType below already specifies this should be an array, why supply a default value here?

Wouldn't this also work fine?

```suggestion
	savedItemSlugs,
```

Reply:
Actually, it is more fitting for it to be required, as it won't work without these props, I removed `[]` default value, and made props required